### DISCUSSION TITLE
solved

### DISCUSSION THREAD

**Discussion by cfrxp** [community/community#133821] [Other Feature Feedback, Questions, & Ideas] [answered]
<closed>

**Comment by LiteBrite82** [comment, marked as answer]
Hi @castipo-hub ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by cfrxp** [reply]
hi sir , can i please request for edit reply to remove my username on this comment, because i just change my username for privacy , sorry for asking that :(

**Comment by cfrxp** [comment]
my account getting back, with support github by mail.

**Comment by MoGlobekiffers** [comment]
perfect
